tck/src/main/resources/jdo/datastoreidentity/org/apache/jdo/tck/pc/company/package.jdo

```diff
@@ -106,6 +106,9 @@ has datastore identity.
             <field name="phoneNumbers" persistence-modifier="persistent">
                 <map key-type="String" value-type="String"/>
             </field>
+            <field name="languages" persistence-modifier="persistent">
+                <collection type="String"/>
```

Comment:
Shouldn't that be `<collection element-type="String"/>`?

Reply:
Good catch!